basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/referential/form/CityPopup.java

```diff
@@ -0,0 +1,50 @@
+package fr.openwide.core.basicapp.web.application.referential.form;
+
+import org.apache.wicket.Component;
+import org.apache.wicket.markup.html.form.CheckBox;
+import org.apache.wicket.markup.html.form.Form;
+import org.apache.wicket.markup.html.form.TextField;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.ResourceModel;
+
+import fr.openwide.core.basicapp.core.business.referential.model.City;
+import fr.openwide.core.basicapp.core.util.binding.Bindings;
+import fr.openwide.core.commons.util.functional.Predicates2;
+import fr.openwide.core.wicket.more.markup.html.basic.ComponentBooleanProperty;
+import fr.openwide.core.wicket.more.markup.html.basic.EnclosureBehavior;
+import fr.openwide.core.wicket.more.markup.html.template.js.jquery.plugins.bootstrap.modal.component.DelegatedMarkupPanel;
+import fr.openwide.core.wicket.more.model.BindingModel;
+
+public abstract class CityPopup extends AbstractGenericListItemPopup<City> {
```

Comment:
This could probably be generic and renamed to `SimpleGenericListItemPopup`, since there's nothing strictly bound to `City` here. So we could use it for other simple `GenericListItem`s.